### PR TITLE
Internal change

### PR DESCRIPTION
**Pull request by jay-shi**
Internal change

### CODE REVIEW

File: proto/google/events/firebase/testlab/v1/data.proto [hunk 1]
```diff
@@ -0,0 +1,6 @@
+syntax = "proto2";
```
Comment (grant) [inline proto/google/events/firebase/testlab/v1/data.proto:1]: APIs should use proto3

File: proto/google/events/firebase/testlab/v1/data.proto [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+package third_party.google_cloudevents.proto.google.events.firebase.testlab.v1;
+
+option java_multiple_files = true;
+option jspb_use_correct_proto2_semantics = true;
```
Comment (grant) [inline proto/google/events/firebase/testlab/v1/data.proto:6]: This is a proto2 only bool.

**Review by grant — CHANGES_REQUESTED** [review]

File: proto/google/events/firebase/testlab/v1/data.proto (hunk 2 above)
Comment (grant) [reply proto/google/events/firebase/testlab/v1/data.proto:6]: Will cause compile tests and thus other tests to fail.

File: proto/google/events/firebase/testlab/v1/data.proto (hunk 1 above)
Comment (jay-shi) [reply proto/google/events/firebase/testlab/v1/data.proto:1]: Sorry I forgot to mention that this PR is only meant for testing. I will revert it now.

**Pull request closed without merging**